src/App.js

```diff
@@ -1,35 +1,19 @@
 import React from 'react'
-const todoList = [
-  {
-    id: 1,
-    title: "Complete assignment list"
-  },
-  {
-    id: 2,
-    title: "Complete assignment map"
-  },
-  {
-    id: 3,
-    title: "Complete assignment objects"
-  }
-];
+import TodoList from './TodoList';
+import AddTodoForm from './AddTodoForm';
 
 function App() {
-  const listItems = todoList.map((number) =>
-    <li key={number["id"].toString()}>
-      {number["title"]}
-    </li >
-  );
   return (
     <div style={{ textAlign: 'left' }}>
       <header>
         <div style={{ textAlign: 'center' }}>
           <h1>Todo List</h1>
         </div>
-        <ul>{listItems}</ul>
+        <TodoList></TodoList>
+        <AddTodoForm></AddTodoForm>
```

Comment:
Nit: Since you're not passing children to those components you can also write it like a self closing html tag:
```
<AddTodoForm />
<TodoList />
 ```
 
 Great job! :100: